scripts/console_container.gd

```diff
@@ -117,35 +120,33 @@ func _show_autocomplete(text: String) -> void:
 		_tooltip_label.set_text("\n".join(autocomplete))
 		_tooltip_label.show()
 
-
 func _on_input_text_changed(text: String) -> void:
-	_autocomplete_index = 0
-	_show_autocomplete(text)
-
+	_show_autocomplete(_autocomplete_prompt if _autocomplete_prompt.length() > 0 else text)
 
 func _cycle_autocomplete(direction: int) -> void:
-	var autocomplete: PackedStringArray = _console.autocomplete_list(_console_input.get_text())
+	var autocomplete: PackedStringArray = _console.autocomplete_list(_autocomplete_prompt)
 	_autocomplete_index = wrapi(_autocomplete_index + direction, 0, autocomplete.size())
-
-	_show_autocomplete(_console_input.get_text())
+	_show_autocomplete(_autocomplete_prompt)
```

Comment:
And are you sure it is necessary to call this method? If we call `set_input_text` then inside this method emitted `_console_input.text_changed` which in turn calls `_show_autocomplete`. It looks like we are calling the same method twice. Correct me if I am wrong.
```suggestion
```

Reply:
You're right. It was needed before, but since we're setting text we can remove it.